.onepipeline.yaml

```diff
@@ -0,0 +1,125 @@
+version: '1'
+
+setup:
+  image: icr.io/continuous-delivery/pipeline/pipeline-base-image:2.12@sha256:ff4053b0bca784d6d105fee1d008cfb20db206011453071e86b69ca3fde706a4
+  script: |
+    #!/usr/bin/env bash
+    set -x
+    GH_TOKEN=$(cat "$WORKSPACE/git-token")
+    OWNER=$(jq -r '.services[] | select(.toolchain_binding.name=="app-repo") | .parameters.owner_id' /toolchain/toolchain.json)
+    REPO=$(jq -r '.services[] | select(.toolchain_binding.name=="app-repo") | .parameters.repo_name' /toolchain/toolchain.json)
+    curl -u ":$GH_TOKEN" https://github.ibm.com/api/v3/repos/$OWNER/$REPO/branches/master/protection -XPUT -d '{"required_pull_request_reviews":{"dismiss_stale_reviews":true},"required_status_checks":{"strict":true,"contexts":["tekton/code-branch-protection","tekton/code-unit-tests","tekton/code-cis-check","tekton/code-vulnerability-scan","tekton/code-detect-secrets"]},"enforce_admins":null,"restrictions":null}'
+    env
+    ls $WORKSPACE
+    oc login $(get_env openshift_cluster) -u $(get_env admin_user) -p $(get_env admin_pw) --insecure-skip-tls-verify=true
+    set +e
+    oc new-project fybrik-system
+    set -e
+    kubectl get po -o wide
+test:
+  abort_on_failure: false
+  image: wcp-ibm-streams-docker-local.artifactory.swg-devops.com/pipelines-tutorial/compliance-baseimage-with-tkn:2.26.1
+  script: |
+    #!/usr/bin/env bash
+    #cd ../"$(load_repo app-repo path)"
+    if [[ "$PIPELINE_DEBUG" == 1 ]]; then
+      trap env EXIT
+      env
+      set -x
+    fi
+    set -x
+    env
+    ls $WORKSPACE
+    oc login $(get_env openshift_cluster) -u $(get_env admin_user) -p $(get_env admin_pw) --insecure-skip-tls-verify=true
+    set +e
+    oc new-project fybrik-system
+    oc project fybrik-system
+    oc delete pipelinerun --all --wait
+    set -e
+    export GH_TOKEN=$(get_env different-git-token)
+    export git_user=$(get_env git-user)
+    . pipeline/source-internal.sh
+    github_workspace=`pwd`/../mesh-for-data run_tkn=$(get_env run_tkn) skip_tests=false bash -x pipeline/bootstrap-pipeline.sh fybrik-system
```

Comment:
I don't think mesh-for-data directory exists after change to fybrik.

Reply:
Perhaps we should remove the one-pipeline file from this commit - currently, this is a separate pipeline, running on ibm cloud, and against our internal fork of the code (which still has a repo name of mesh-for-data).

Regardless, I think I can update the repo name and 'fix' this code to reference fybrik